lib/NfcoreTemplate.groovy

```diff
@@ -32,6 +32,21 @@ class NfcoreTemplate {
         }
     }
 
+    //
+    //  Warn if using custom configs to provide pipeline parameters
+    //
+    public static void warnParamsProvidedInConfig(workflow, log) {
+        if (workflow.configFiles.size() > 1) {
+            log.warn "=============================================================================\n" +
+                "  Multiple config files detected!\n" +
+                "  Please provide pipeline parameters via the CLI or Nextflow '-params-file' option.\n" +
+                "  Custom config files including those provided by the '-c' Nextflow option can be\n" +
+                "  used to provide any configuration except for parameters.\n\n" +
+                "  Docs: https://nf-co.re/usage/configuration#custom-configuration-files\n" +
```

Comment:
Bit late so not my best, but wonder if this is _slightly_ clearer? Gives some reasoning..

```suggestion
                "  Multiple config files detected!\n" +
                "  Please provide pipeline parameters via the CLI or Nextflow '-params-file' option.\n" +
                "  Nextflow config files (eg. using '-c') do not overwrite params properly, but can\n" +
                "  be used for other config scopes.\n\n" +
                "  Docs: https://nf-co.re/usage/configuration#custom-configuration-files\n" +
```